src/VirtualClient/VirtualClient.Main/profiles/QUAL-GPU-DCGMI-DIAG.json

```diff
@@ -0,0 +1,37 @@
+﻿{
+  "Description": "DCGMI Monitor",
+  "Parameters": {
+    "Level": "4",
+    "Username": "testuser",
+    "CudaVersion": "11.6",
+    "DriverVersion": "510",
+    "LocalRunFile": "https://developer.download.nvidia.com/compute/cuda/11.6.0/local_installers/cuda_11.6.0_510.39.01_linux.run"
```

Comment:
I know that this seems like a good idea to let folks change the version of Cuda and the Run file. However, we cannot firmly say that the user can put any version he/she wants here. Factually, we will NOT have had a chance to test this for production quality given the user overrides the version. I think we should remove the CudaVersion and LocalRunFile from the global parameters. Users can always create a profile of their own if they want to change versions from what we have confirmed as Production-ready. I just do not want the CRC team to own a responsibility or SLA to support this. There are just too many variables involved in that. So, let's just remove these from the global parameters and put them in the dependencies directly.